src/input.service.ts

```diff
@@ -264,8 +272,16 @@ function isForForm(direction: Direction, selected: Element): boolean {
 @Injectable()
 export class InputService {
 
-  private inputPane = Windows.UI.ViewManagement.InputPane.getForCurrentView();
-  public keyboardVisible = new BehaviorSubject(false);
+  /**
+   * Inputpane and boolean to indicate whether it's visible
+   */
+  private inputPane = (<any>window).Windows ? Windows.UI.ViewManagement.InputPane.getForCurrentView() : null;
+  public keyboardVisible = (<any>window).Windows ? new BehaviorSubject(false) : null;
```

Comment:
Sure. Fixed. I used null b/c inputPane would be null so we need to check for null in that case anyway.